building/javascript-sdk.md

```diff
@@ -201,7 +203,13 @@ const tbl = await tl.connect({ signer, network: "testnet" });
 
 ### Creating Tables
 
-Like most relational database systems, Tableland requires the user to create tables for storing, querying, and relating data. This is done via the `create` function. The `create` function takes a plain SQL statement string. Most valid SQL _constraints_ are supported, and the following data types are currently [supported](https://github.com/textileio/go-tableland/blob/main/pkg/parsing/query\_validator.go):
+Like most relational database systems, Tableland requires the user to create tables for storing, querying, and relating data. This is done via the `create` function. The `create` function takes 3 arguments: 
+1. The chainId of the chain you specified while connecting.
```

Comment:
When we make this optional, let's also document that it is optional here. Note also that that means we have to change the structure of the function signature. We can't have an optional as the first argument.